dashboard.csss-ci.com/server/api/models/Households_model.php

```diff
@@ -97,6 +166,11 @@ public function age0($where) {
                         ->result();
     }
 
+    /**
+     * Returns all records with desired age association
+     *
+     * @return Array containing rows with desired age association
+     */
     public function ages0() {
         return $this->db->select_sum('age0')
                         ->from($this->table)
```

Comment:
in addition to the above problems, these should also mention something about ages